blocks/right-rail-advanced-block/layouts/right-rail-advanced/default.scss

```diff
@@ -41,7 +41,12 @@ body {
 
 
 @media (max-width: 64rem) {
+  .row.advanced-grid {
+    grid-template-columns: auto;
```

Comment:
is this new for the pr? thought it was just storybook?

Reply:
It's need for storybook - and doesn't affect output